### DISCUSSION TITLE
X86/64固件不显示在线用户数，但是dhcp显示正常，找不到问题所在，求解决方法。

### DISCUSSION THREAD

**Discussion by chenxuan5858** [immortalwrt/immortalwrt#1145] [Q&A] [answered]
### Describe the bug

X86/64固件不显示在线用户数，但是dhcp显示正常，找不到问题所在，求解决方法。

### ImmortalWrt version

r28542-899c50c971

### ImmortalWrt release

SNAPSHOT

### ImmortalWrt target/subtarget

x86/64

### Device

X64

### Image kind

Self-built image

### Steps to reproduce

![1fae0518e994714f4cedfe33bae28f9](https://github.com/immortalwrt/immortalwrt/assets/33675890/9eb8d637-8799-4cce-ae1f-b7fca3a45ffa)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by chenxuan5858** [comment]
有大佬知道是什么问题吗？

**Comment by xlighting2017** [comment]
`cat /proc/net/arp` 会出来什么

盲猜 你的 LAN侧 不是  br-lan

**Reply by chenxuan5858** [reply]
cat /proc/net/arp 会出来
Lan确实不是br-lan，找不到这一项的配置入口

**Reply by xlighting2017** [reply]
贴一下 会出来什么 内容

这个统计，是统计  br-lan 接口上面的 ARP 地址数， 你没这个接口， 自然数量=0

**Reply by chenxuan5858** [reply]
![Uploading 微信图片_20240104174210.png…]()

**Reply by chenxuan5858** [reply]
我当时也估计是这样，但是我有vlan，br-lan已经不存在了，所以想请教是怎么修改配置

**Reply by chenxuan5858** [reply]
![Uploading 微信图片_20240104174210.png…]()

**Reply by chenxuan5858** [reply]
![Uploading 20240104174210.png…]()

**Reply by chenxuan5858** [reply]
![20240104174210](https://github.com/immortalwrt/immortalwrt/assets/33675890/c06c5a46-2efd-4716-8724-910b92eb00f8)

**Reply by chenxuan5858** [reply]
upnp也出过问题，但我能找到配置，修改后正常了，这个我没找到配置的地方，所以来请教一下

**Reply by xlighting2017** [reply, marked as answer]
`vi /usr/share/rpcd/ucode/luci`
![image](https://github.com/immortalwrt/immortalwrt/assets/25970065/413a0c62-435d-4356-9c0b-4e627d347dbd)

这里写死了是  br-lan,  你要统计哪个接口，就改成对应的字符串

如果升级版本，应该还需要再手动修改

**Reply by chenxuan5858** [reply]
感谢！我修改一下，非常感谢！
